lib/ecto_boot_migration.ex

```diff
@@ -209,12 +221,6 @@ defmodule EctoBootMigration do
   def log(msg), do: log(msg, debug?())
   def log(msg, true), do: IO.puts("[EctoBootMigration] #{msg}")
 
-  @doc """
-  this prevents pre-mature return
-  that could cause the main app to exit, because the repos were not 100% shutdown
-  """
-  def log(_, false), do: Process.sleep(1)
```

Comment:
As far as I can tell, this is never called if you have verbose output enabled (which we do), and explicitly waiting on each repo to return a `DOWN` message should obviate the need for this.